modules/distribution/src/main/config/fluo-env.sh

```diff
@@ -35,7 +35,11 @@ export FLUO_CONN_PROPS="${FLUO_CONN_PROPS:-${conf}/fluo-conn.properties}"
 ## Fluo log4j configuration
 export FLUO_LOG4J_CONFIG="${FLUO_LOG4J_CONFIG:-${conf}/log4j.properties}"
 ## Java options for Fluo command
-JAVA_OPTS=("-Dlog4j.configuration=file:${FLUO_LOG4J_CONFIG}")
+JAVA_OPTS="-Dlog4j.configuration=file:${FLUO_LOG4J_CONFIG}"
```

Comment:
No longer need this first line here, since it is overridden two lines lower now. :)